CHANGELOG.md

```diff
@@ -85,7 +85,7 @@ __New Features__
           props.children,
         );
       }, 
-      $FancyBorderConfig, // ignore: undefined_identifier
+      _$FancyBorderConfig, // ignore: undefined_identifier
```

Comment:
Would we not want to update this one to keep the changelog history that, in version 3.8.0, factory configs were public and then a later changelog entry will say they were changed to private? Or does it not really matter?

Reply:
That's a good question. It seemed fine to just keep it all consistent so there's only one way to do functional components, but I could see it going either way! To be safe I undid the change!